src/main/java/com/greengroupecommerce/controllers/StripeController.java

```diff
@@ -0,0 +1,43 @@
+package com.greengroupecommerce.controllers;
+
+import com.stripe.Stripe;
+import com.stripe.model.checkout.Session;
+import com.stripe.param.checkout.SessionCreateParams;
+
+import lombok.Value;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.servlet.view.RedirectView;
+
+@Controller
+@RequestMapping("/call-checkout")
+public class StripeController {
+
+    @PostMapping
+    public RedirectView createCheckoutSession() throws Exception {
+        @Value("${stripe.api.secretKey}")
+        final String secretKey;
+
+        Stripe.apiKey = secretKey;
+
+        String YOUR_DOMAIN = "http://localhost:8080";
```

Comment:
I noticed you mapped this to the dev port, is it gonna be an issue in prod?

Reply:
Yes, forgot to change.